aioredis/connection.py

```diff
@@ -715,9 +756,10 @@ async def on_connect(self):
         # if username and/or password are set, authenticate
         if self.username or self.password:
             if self.username:
-                auth_args = (self.username, self.password or "")
+                auth_args: Tuple[str, ...] = (self.username, self.password or "")
```

Comment:
Intentional `...`?

Reply:
Indicates a tuple of any length. Given the alternative form on line 762, it's either this or `Union[Tuple[str], Tuple[str, str]]`, which is technically more accurate, but not enough benefit to warrant the awkwardness.

Reply:
it's a dumb nitpick of mine, but I'd like the more verbose `Union[Tuple[str], Tuple[str, str]]`. Thanks for being patient with me.